setup.cfg

```diff
@@ -48,3 +48,7 @@ install_requires =
     colcon-zsh; sys_platform != 'win32'
 packages = find:
 zip_safe = true
+
+[options.extras_require]
+recommended =
+    colcon-override-check
```

Comment:
I think it's good either way but is `extras_require[recommended]` an accepted community norm or is this something that we're trailblazing based on distro packaging conventions?

Reply:
I've found no precedent. Fundamentally, the purpose of `extras_require` is to list optional or recommended dependencies that aren't strictly required, so we're not very far out by using it like this. That said, I found no uses of the name `recommended`, and this is also essentially a Python metapackage, for which there is little precedent to begin with.